app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        new_file = command.split()
+        if command[0] == "cp" and new_file[1] != new_file[2]:
```

Comment:
The condition `command[0] == 'cp'` is incorrect. You should check `new_file[0] == 'cp'` instead, as `new_file` is the result of `command.split()` and contains the split parts of the command.